src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/wrapper/SolicitorCaseDataKeysWrapper.java

```diff
@@ -9,4 +9,5 @@ public class SolicitorCaseDataKeysWrapper {
     private String solicitorReferenceKey;
     private String solicitorNameKey;
     private String solicitorEmailKey;
+    private Boolean solicitorIsNotDigitalKey;
```

Comment:
Can this be a primitive boolean rather than an object? What happens if it is null?